### PR TITLE
button for reset, c.1uF0603

### PR DESCRIPTION
**Pull request by QingmuDeng**

### CODE REVIEW

File: schematic/formula.dcm [hunk 1]
```diff
@@ -80,6 +84,10 @@ $CMP C_0.1uF
 D Capacitor, 0.1uF, 50V, 5%, 0805
 $ENDCMP
 #
+$CMP C_0.1uF_0603
+D CAP CER 0.1UF 16V X7R 0603
```
Comment (ljordan51) [inline schematic/formula.dcm]: i'd appreciate if you follow the format of every single other cap
we should probably add dielectric ratings to them all, but you can just tag it onto the end for now

File: schematic/formula.lib [hunk 2]
```diff
@@ -553,6 +553,25 @@ X P3 3 -250 -100 150 R 50 50 1 1 P
...
 ENDDEF
 #
+# CONN_01x04
+#
+DEF CONN_01x04 J 0 40 Y Y 1 F N
+F0 "J" 50 200 50 H V C CNN
+F1 "CONN_01x04" 0 300 50 H V C CNN
+F2 "Pin_Headers:Pin_Header_Straight_1x04_Pitch2.54mm" 0 200 50 H I C CNN
+F3 "https://katalog.we-online.de/em/datasheet/6130xx11121.pdf" 0 200 50 H I C CNN
+F4 "732-5317-ND" 250 400 50 H I C CNN "MPN"
```
Comment (ljordan51) [inline schematic/formula.lib]: please fix the order of the fields to match every other component

File: schematic/formula.lib [hunk 3]
```diff
@@ -3992,6 +4036,26 @@ X PC9 99 800 -1850 200 L 50 50 2 1 B
...
 ENDDEF
 #
+# SW_Push_SPST_NO
+#
+DEF SW_Push_SPST_NO SW 0 40 Y Y 1 F N
+F0 "SW" 100 150 50 H V C CNN
+F1 "SW_Push_SPST_NO" 50 -100 50 H V C CNN
+F2 "footprints:SW_B3U-1000P_4.2x1.7mm" 0 50 50 H I C CNN
+F3 "https://omronfs.omron.com/en_US/ecb/products/pdf/en-b3u.pdf" 0 50 50 H I C CNN
+F4 "SW1020CT-ND" 200 250 50 H I C CNN "MPN"
```
Comment (ljordan51) [inline schematic/formula.lib]: please fix the order of the fields to match every other component

File: schematic/formula.dcm [hunk 4]
```diff
@@ -68,6 +68,10 @@ $CMP CONN_01X03
 D Connector header through hole 1x3 0.100" (2.54mm)
 $ENDCMP
 #
+$CMP CONN_01x04
+D CONN HEADER 4 POS 2.54
```
Comment (ljordan51) [inline schematic/formula.dcm]: please change description to match format of 1x3 header

File: schematic/formula.dcm (hunk 1 above)
Comment (QingmuDeng) [reply schematic/formula.dcm]: the thing is there is a .1uF that is 0805. 0603 is just for distinction.

File: schematic/formula.lib (hunk 2 above)
Comment (QingmuDeng) [reply schematic/formula.lib]: I fixed everything else but will send a pull request along once I'm done with a USB part

File: schematic/formula.dcm (hunk 1 above)
Comment (ljordan51) [reply schematic/formula.dcm]: Sorry I wasn't more clear, I like the way you named it to distinguish, but your description (line 88 starting wtih D) doesn't match the other capacitors in the library

**Pull request closed without merging**